### PR TITLE
Update to latest ubi8 base image

### PR DESCRIPTION
**Pull request by rickjstevens**
Signed-off-by: Rick Stevens <rstevens@us.ibm.com>

### CODE REVIEW

**Comment by rickjstevens** [comment]
CI toolchain was failing due to vulnerability advisor errors. Moving to latest ubi8/ibmjre base image resolved those issues.

**Comment by msargentibm** [comment]
Does the version need bumped again too?

**Comment by rickjstevens** [comment]
> 

I was thinking we would stay with current version given underlying version of the open source cql framework projects has not changed. Only change we're making in this PR is to run on newer versions of ubi8 and Java. Is there a compelling reason to bump version number when the underlying functionality of the code in question should not be changing?

**Comment by msargentibm** [comment]
> Is there a compelling reason to bump version number when the underlying functionality of the code in question should not be changing?

Only if this triggers a new build and release to github. I wasn't sure if that was the case or not.

**Comment by rickjstevens** [comment]
> > Is there a compelling reason to bump version number when the underlying functionality of the code in question should not be changing?
> 
> Only if this triggers a new build and release to github. I wasn't sure if that was the case or not.

Per standup discussion - will update version to a SNAPSHOT based on current version of upstream project we're based on (1.5.6).

**Review by jillrdoty — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # fetch basic image
-FROM us.icr.io/wh-common-rns/base-images/ubi8-ibmjre:20220124.2042
+FROM us.icr.io/wh-common-rns/base-images/ubi8-ibmjre:20220317.0842
 
 WORKDIR /
 
-COPY target/cql-translation-server-1.5.1-2-jar-with-dependencies.jar cql-translation-server-jar-with-dependencies.jar
+COPY target/cql-translation-server-1.5.6-SNAPSHOT-jar-with-dependencies.jar cql-translation-server-jar-with-dependencies.jar
```
Comment (tabishop) [inline Dockerfile]: Wonder if you could try "COPY target/*-jar-with-dependencies.jar ..." so you don't have to update it when the version changes or a release is done.

**Review by tabishop — APPROVED** [review]
1 minor comment, wouldn't hold up the PR for it tho

File: Dockerfile (hunk 1 above)
Comment (rickjstevens) [reply Dockerfile]: Good suggestion - simple way to avoid dual maintenance when updating to new versions in the future.

**Pull request merged**